test/lit.cfg

```diff
@@ -151,3 +151,8 @@ else:
 # POSIX runtime feature
 if config.enable_posix_runtime:
   config.available_features.add('posix-runtime')
+
```

Comment:
@delcypher I modified the configuration in this way. If there's a better way in your opinion, please let me know.